Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -564,6 +580,7 @@ final class LayerTests: XCTestCase {
         ("testRNN", testRNN),
         ("testLSTM", testLSTM),
         ("testFunction", testFunction),
+        ("testGRU", testGRU),
```

Comment:
Could you please move this above `("testFunction", testFunction),` since the `testGRU` function is defined above `testFunction`?